third_party/amd/backend/compiler.py

```diff
@@ -71,6 +72,36 @@ def hash(self):
         return hashlib.sha256(key.encode("utf-8")).hexdigest()
 
 
+class HIPAttrsDescriptor(AttrsDescriptor):
+    __slots__ = ("pointer_range_32")
```

Comment:
Could you put a comment in the above explaining what this means, how we deduce it, and what it is for?